pay-with-amazon.js

```diff
@@ -159,9 +161,7 @@ function PayWithAmazon (opts) {
     this.init();
   } else {
     document.write('<script src="'
-      + 'https://static-na.payments-amazon.com/OffAmazonPayments'
-      + '/us/sandbox/js/Widgets.js?sellerId='
-      + this.config.sellerId
+      + this.getAssetPath()
```

Comment:
Since we're not going to go over the column limit anymore, this can be

```js
document.write('<script src="' + this.getAssetPath() + '"></script>');
```